### DISCUSSION TITLE
Kubernetes Deployment

### DISCUSSION THREAD

**Discussion by Pacerino** [hcengineering/platform#5199] [General]
Hello!

I came across the issue #1042 where it is written that the Kubernetes deployment was pulled privately. I wanted to ask if there is a specific reason for this?

I would like to deploy Huly on our K8 cluster.

Greetings

**Comment by aplatoff** [comment]
Hi @Pacerino,

I guess this is quite old issue. We're deploying Huly to Kubernetes both for staging and production. And I think the process is open and visible at least in our GitHub CI. Maybe @haiodo can shed more light on this question :)

Kind regards,
Andrey

**Reply by aplatoff** [reply]
Oh I just take a look and it seems it's still private. Please let us discuss this issue internally and we'll get back to you shortly. Thank you for open this discussion

**Reply by Pacerino** [reply]
Thank you for your reply! I'm looking forward for your reply!

**Reply by AlwaysNoobCoder** [reply]
I want this too

**Reply by aplatoff** [reply]
Hi guys!

We will make them public soon, since we're finishing making all Huly public, except some integrations, but they will be available as Docker containers and ready for k8s deployment.

Thank you!

**Reply by Pacerino** [reply]
Hey @aplatoff

Could you give me a time range when we can expect the release?

Cheers :)

**Reply by aplatoff** [reply]
@aonnikov Hey Alex, I think you did something in this regard. Please could you shed some light on the topic?

**Reply by Pacerino** [reply]
@aplatoff Hey there!

I haven't done anything about it yet. I would like to deploy the project on our K8 cluster. I had seen that there used to be a Helm repo etc., but it was pulled privately. See the issue mentioned above.

**Reply by aonnikov** [reply]
@Pacerino hey!
I'm not aware about any Helm chart for Huly deployment. But I recently pushed example Kubernetes configuration to the huly-selfhost repository. 

You can find it here:
https://github.com/hcengineering/huly-selfhost/tree/main/kube

**Reply by aplatoff** [reply]
Hi @Pacerino, I'm asking for help another Alex -- @aonnikov, he did something to make k8s deployment public recently :)

**Reply by Pacerino** [reply]
Oh, i haven't seen that you mentioned the other Alex :D
Anyways, thanks for the selfhosted repo. I'll take a look into it!
